cmd/guacgql/cmd/server.go

```diff
@@ -80,6 +80,11 @@ func startServer(cmd *cobra.Command) {
 		logger.Errorf("unable to initialize graphql server: %v", err)
 		os.Exit(1)
 	}
+	metric, err := cli.SetupPrometheus(ctx, logger, "guacgql")
+	if err != nil {
+		logger.Fatalf("Error setting up Prometheus: %v", err)
+	}
```

Comment:
You need to check if `metric` is `nil` since `cli.SetupPrometheus` can `return nil, nil`

Reply:
I agree. Thanks! I took care of it.